### PR TITLE
Adapt User Guide to VirusTracker

### LINKED ISSUE #85: Adapt User Guide to VirusTracker
Adapt the user guide for AB3 for the use of VirusTracker.

### PR DESCRIPTION
**Pull request by siangernlow**
Close #85

### CODE REVIEW

**Review by KohHanMing — APPROVED** [review]
Other than the list people command, everything looks good :)

File: docs/UserGuide.md [hunk 1]
```diff
@@ -215,5 +227,5 @@ Action | Format, Examples
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
 **Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
 **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
+**List** | `list l/KEYWORD` <br> e.g., `list people`, `list stats`
```
Comment (hopinxian) [inline docs/UserGuide.md:230]: I think that to match prefix, the e.g. would be sth like list l/people, list /stats. Please check on this.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -150,7 +159,7 @@ Format: `exit`
 
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+AddressBook data saves in the hard disk automatically after any command that changes the data. There is no need to save manually.
```
Comment (hopinxian) [inline docs/UserGuide.md:162]: Might want to refer to this as Virus Tracker instead of AddressBook.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -138,7 +147,7 @@ Examples:
 
 ### Clearing all entries : `clear`
 
-Clears all entries from the address book.
+Clears all entries in the current list view from VirusTracker.
```
Comment (hopinxian) [inline docs/UserGuide.md:150]: Might consider changing to 'Clears all entries from Virus Tracker' to match description in line 42.

**Review by hopinxian — CHANGES_REQUESTED** [review]
LGTM after minor changes.

**Review by hopinxian — APPROVED** [review]
LGTM

**Pull request merged**